shared/trainer/defaults.yaml

```diff
@@ -124,6 +125,7 @@ dis:
   m:
     <<: *default-dis
     multi_level: false
+    architecture: OmniDiscriminator # can be [default | OmniDiscriminator]
```

Comment:
can we change default to base or something else ?

Reply:
Sure!